### PR TITLE
amqplib: expose Option and Reply properties again (bugfix)

### PR DESCRIPTION
**Pull request by abreits**
Forgot to export the Option, Reply and Message interfaces from within the 'amqplib' and 'amqplib/callback_api' modules. ( Oops! )

Fixed and added simple tests to check for them in the future. 
It is now compatible again with the original amqplib syntax.

### CODE REVIEW

**Comment by abreits** [comment]
Also found out just now that the Typescript compiler complains about the properties with type Object.
The example code:

    var amqpMessage: amqp.Message;
    amqpMessage.properties.contentType === "application/json";

generated an error during compile, changing the type to 'any' fixes it.I have not found a better type definition that can narrow it down to a `{key: value}` list ( `{[id: string]: any}` does not work).

**Comment by vvakame** [comment]
*amqplib/amqplib.d.ts*

to authors(@mnahkies @abreits). could you review this PR?
:+1: or :-1:?

check list

* [ ] pass the Travic-CI test?

**Comment by abreits** [comment]
Since I am the author of the request:
:+1:
However the original author of the library must of course also agree with it.

File: amqplib/amqplib-tests.ts [hunk 1]
```diff
@@ -51,3 +59,9 @@ amqpcb.connect("amqp://localhost", (err, connection) => {
         });
     }
 });
+
+// test callback api properties
+var amqpcbMessage: amqp.Message;
```
Comment (mnahkies) [inline amqplib/amqplib-tests.ts]: I think you meant to reference amqpcb.Message, etc here?

**Comment by abreits** [comment]
To (@mnahkies): you are correct (stupid copy paste error). I fixed the test, so now :+1: ?

**Comment by mnahkies** [comment]
var amqpcbAssertExchangeOptions: amqp.Options.AssertExchange;
    var anqpcbAssertExchangeReplies: amqp.Replies.AssertExchange;

Require changing too - sorry to be a pain. Otherwise is looking good.

**Comment by abreits** [comment]
No problem, I've added the changes (should have seen those myself, still not fully awake I guess...).
So now :+1: ?

**Comment by mnahkies** [comment]
:+1:  thanks

**Comment by vvakame** [comment]
thanks mate!

**Pull request merged**